src/libs/actions/IOU.js

```diff
@@ -111,8 +111,34 @@ function createIOUSplit(params) {
         });
 }
 
+/**
+ * Settles an IOU Report and then retrieves the iou and chat reports to trigger updates to the UI.
+ */
+function settleIOUReport({
+    chatReportID, reportID, paymentMethodType,
+}) {
+    Onyx.merge(ONYXKEYS.IOU, {loading: true, error: ''});
+    API.PayIOU({
+        reportID,
+        paymentMethodType,
+    })
+        .then((response) => {
+            if (response.jsonCode !== 200) {
+                throw new Error(`Error while settling IOU: ${response.message}`);
+            }
+        })
+        .then(() => fetchChatReportsByIDs([chatReportID]))
```

Comment:
Does this API call need to finish before the next one, or can they happen in parallel?

Reply:
Nope. Presumable I would move them out of the then promise chain and call both functions after checking for the [response code here](https://github.com/Expensify/Expensify.cash/pull/2394/files/918b7b612c9c51af8aa8b6ede42741b9974cf8a1#diff-17f969ae17c2a0f476494263758dbe0a535432f496b805345925e648d605915cR126-R128)?

Reply:
Yep! Just call them both, one right after the other.